includes/widgets/tabs.php

```diff
@@ -117,6 +128,19 @@ protected function register_controls() {
 			]
 		);
 
+		// Check if TabsV2 experiment is enabled.
+		if ( Plugin::$instance->experiments->is_feature_active( 'tabs-v2' ) ) {
```

Comment:
Make it generic, add a method `deprecation_message` that is based on the `deprecated` config.
```suggestion
		$this->deprecation_message();
```